ansible/files/etc/httpd/conf.modules.d/00-ssl.conf

```diff
@@ -1 +0,0 @@
-LoadModule ssl_module modules/mod_ssl.so
```

Comment:
Removing the shared object file for the SSL module isn't needed for anythibg else?  I guess as long as the app deploys and works then this is probably fine.